package/gocryptfs/package

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+# Copyright (c) 2020 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+pkgnames=(gocryptfs)
+pkgdesc="An encrypted overlay filesystem written in Go."
+url="https://nuetzlich.net/gocryptfs/"
+_srcver=v2.0-beta2
+pkgver=$_srcver-1
+timestamp=2021-03-22
+section=utils
+maintainer="plan5 <30434574+plan5@users.noreply.github.com>"
+license=MIT
+
+depends=(fuse-utils)
+
+makedepends=(build:golang build:git)
+
+image=base:v1.4
+
+source=(
+    https://github.com/rfjakob/gocryptfs/archive/$_srcver.zip
+)
+
+sha256sums=(
+    3d66368cfc79a300de5a22fe01788d0c702f9107731db63fcd8850157d105cc0
+)
+
+prepare() {
+    # Official build script runs compiled binary to show version. 
```

Comment:
```suggestion
    # Official build script runs compiled binary to show version.
```

Reply:
Can't merge because I already removed the trailing space on my branch before seeing this.
So it should be taken care of :)